code/modules/mob/mob_stat.dm

```diff
@@ -245,6 +245,11 @@
 		var/ETA = SSshuttle.emergency.getModeStr()
 		if(ETA)
 			tab_data[ETA] = GENERATE_STAT_TEXT(SSshuttle.emergency.getTimerStr())
+	if (!isnewplayer(src) && SSautotransfer.can_fire)
+		if (SSautotransfer.required_votes_to_leave && SSshuttle.canEvac())
```

Comment:
SSshuttle.canEvac returns a string, which means its always true. At the very least it shouldn't show the players voting to leave until the shuttle is ready to be called (as early in the round you could meta the number of people). The more restrictions the better, but that one is my minimum.

Reply:
fixed with the same modification I gave in my original code and also almost the same comment I had next to it warning that `canEvac()` always returns a value. 

Read the comments!